python_scripts/trees.py

```diff
@@ -182,13 +183,13 @@ def plot_decision_function(X, y, clf, ax=None):
 )
 
 # %% [markdown]
-# Indeed, it is not a surprise. We earlier saw that a single feature will not
-# help separating the three species. However, from the previous analysis we
-# saw that using both features should be useful to get fairly good results.
+# Indeed, it is not a surprise. We saw earlier that a single feature will not
+# be able to separate all three species. However, from the previous analysis we
+# saw that by using both features we should be able to get fairly good results.
 # Considering the mechanism of the decision tree illustrated above, we should
```

Comment:
'Considering the mechanism of the decision tree illustrated above' - this is a little confusing to me. mechanism = making splits ? If so this might not be necessary.

Reply:
https://github.com/lucyleeow/scikit-learn-mooc/pull/3